pkg/platform/abstract/platform.go

```diff
@@ -1236,6 +1240,22 @@ func (ap *Platform) validateNodeSelector(functionConfig *functionconfig.Config)
 	return nil
 }
 
+func (ap *Platform) validatePriorityClassName(functionConfig *functionconfig.Config) error {
+
+	if functionConfig.Spec.PriorityClassName == ap.Config.Kube.DefaultFunctionPriorityClassName {
```

Comment:
thought it is pretty readable - perhaps you want to leave some comments

e.g.:
1. function uses default class name
2. look for class name in list of valid class names